auth/src/express/middleware/route-service.ts

```diff
@@ -38,29 +38,29 @@ export default class RouteService extends RouteHandler {
             return;
         }
         response.locals['newToken'] = result.result?.token;
-        this.sendResponse(true, 'Authentication successful!', response);
+        this.sendResponse(true, undefined, response);
```

Comment:
Maybe remove the 2. parameter or make it ioptional?

Reply:
The second parameter is optional, therefore I changed the calls to `undefined` :D
But I can change the order, so in the most cases only two parameters are passed.

Reply:
This is better